feat: add files to skip usdchecker

beersandrew opened the pull request:
- added files to be skipped
- added skipping logic to the action file

beersandrew committed:
feat: add files to skip usdchecker

- added files to be skipped
- added skipping logic to the action file

pablode commented on .github/workflows/PR-validation.yml:
```diff
@@ -22,6 +22,7 @@ jobs:
       with:
         files: |
           **/*.usd*
+        files_ignore_from_source_file: .skip-usdchecker-files
```
Nice solution, wasn't aware of it! 👍

Since the file is only used by the GitHub Actions workflow, perhaps we could move it to the `.github` folder to not pollute the root dir? We could then name it `skip-usdchecker-files.txt`.

pablode commented on .skip-usdchecker-files:
```diff
@@ -0,0 +1,11 @@
+test_assets/NormalsTextureBiasAndScale/NormalsTextureBiasAndScale.usdz
+test_assets/NormalsTextureBiasAndScale/NormalsTextureBiasAndScale.usda
```
These should not be ignored - usdchecker only emits a warning and the exit code is still 0.

pablode commented on .skip-usdchecker-files:
```diff
@@ -0,0 +1,11 @@
+test_assets/foundation/stage_composition/references_prim/reference_prim_in_same_file.usda
+test_assets/foundation/stage_composition/references_prim/reference_prim_in_other_file.usda
+test_assets/foundation/stage_configuration/invalid_defaultPrim/invalid_defaultPrim.usda
+test_assets/foundation/stage_configuration/framesPerSecond/framesPerSecond_0.usda
+test_assets/foundation/stage_configuration/framesPerSecond/framesPerSecond_-1.usda
```
Although usdchecker fails for these two assets, it does so by crashing in the parsing stage - implying a bug in USD. (Tested with USD v23.05.) Perhaps it would be best to report this as a bug to the OpenUSD repo, if it still persists in v23.08 (to be released today).

Regardless, I don't think these are intended failure cases.

beersandrew replied:
Okay, I'll test this in v23.08 and if it's still happening, report it there

pablode commented on .skip-usdchecker-files:
I think `test_assets/foundation/stage_configuration/upAxis/upAxis_invalid.usda` is another asset we need to exclude.

beersandrew committed:
fix: update checker file

- updated it's location to be out of the root dir
- updated contents to exclude warnings and crashes, including upAxis_invalid
- updated filename to include .txt for clarity

beersandrew committed:
fix: update path to checker file

pablode commented:
Thanks for the PR! 😃
